static/js/stream_settings_ui.js

```diff
@@ -431,8 +433,24 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+export function render_on_empty_streams(is_empty) {
+    if (is_empty) {
+        $(".nothing-to-show").show();
+        if (is_subscribed_stream_tab_active()) {
+            $(".view_all_stream_link_container").show();
+            $(".create_stream_link_container").hide();
+        } else {
+            $(".view_all_stream_link_container").hide();
+            $(".create_stream_link_container").show();
+        }
+        return;
+    }
+    $(".nothing-to-show").hide();
```

Comment:
I think a better structure here would be to have `hide` code followed by return statement inside `if` block and then followed by the code to show the text.